src/pages/therapies-description/index.js

```diff
@@ -151,7 +151,7 @@ export const DetailsTherapies = () => {
               <li>
                 {therapies.attributes?.url_agendamento == null && (
                   <>
-                    <div className="inscriptionIcon">
+                    <div className="inscription-icon">
```

Comment:
O que esse ícone faz? É para se inscrever no evento? Se for, sugiro outros nomes, pois acho que `inscription` não traduz corretamente para `inscrição`:

```suggestion
                    <div className="sign-up-icon">
```

```suggestion
                    <div className="register-icon">
```

```suggestion
                    <div className="attend-icon">
```

```suggestion
                    <div className="join-icon">
```